server/src/test/java/io/crate/integrationtests/ObjectColumnTest.java

```diff
@@ -115,6 +115,16 @@ public void testAddColumnToIgnoredObject() throws Exception {
         assertEquals(detailMap, response.rows()[0][1]);
         assertEquals(4.8d, response.rows()[0][2]);
         assertEquals("1982-01-01", response.rows()[0][3]);
+
+        // Additional check to get zero rows after deletion
```

Comment:
Could you create a dedicated test case for this?

Reply:
Done. It similar to `testAddColumnToIgnoredObject` just simplified it to insert only one non-indexed column and select only it before and after deletion